tests/mock_smtp.py

```diff
@@ -70,7 +70,7 @@ def loop(self):
         exit.
         """
         while len(asyncore.socket_map):
-            asyncore.loop(timeout=1, use_poll=True)
+            asyncore.loop(timeout=0, use_poll=True)
```

Comment:
It made the "bad file descriptor" errors appear to go away on my machine, I will revert it back to 1 (I was not aware of the reason for it being at 1).